src/feedspora/feedspora_runner.py

```diff
@@ -330,6 +292,108 @@ def test_output(self, text):
 
         return self.output_test(output)
 
+    # pylint: disable=too-many-branches
+    def shorten_url(self, the_url):
+        '''
+        Apply configured URL shortener (if present) to the provided link and
+        return the result.  If anything goes awry, return the unmodified link.
+        '''
+        to_return = the_url
+        # Default
+        short_options = {'timeout': 3}
+        short_options.update(self._url_shortener_opts)
+        if the_url and self._url_shortener and (self._url_shortener != 'none'):
```

Comment:
the parentheses are not required :-)

Reply:
Just when you think you have them all removed...
I'll take care of it.

Reply:
That fix is in now.